salt/_pillar/metalk8s.py

```diff
@@ -170,6 +170,7 @@ def ext_pillar(minion_id, pillar, bootstrap_config):  # pylint: disable=unused-a
             "networks": _load_networks(config),
             "metalk8s": metal_data,
             "proxies": config.get("proxies", {}),
+            "kubeApiServer": config.get("kubeApiServer", {}),
```

Comment:
Can we do something more future-proof, like:

```yaml
kind: BootstrapConfiguration
# [...]
kubernetes:
  apiServer:
    featureGates:
      ephemeralContainers: "true"
```

So we can add fields easily to `kubernetes` and keep it well-contained (also set default values, if needed, and share values across core K8s components, if that's something).

Reply:
Sure I will

Reply:
done